src/RPtest.sch

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema xmlns:sch="http://purl.oclc.org/dsdl/schematron" queryBinding="xslt2"
+    xmlns:sqf="http://www.schematron-quickfix.com/validator/process">
+    <pattern>
+        <rule context="contrib"
+            <report test="collab" role="warning" id="test-collab-presence">This article includes a group author - ensure it displays correctly.</report>
+        </rule>
+    </pattern>
+    
+    <pattern>
+        <rule context="app-group"
+            <report test="." role="warning" id="test-appendix-presence">This article includes one or more appendices - ensure these display correctly.</report>
+        </rule>
+    </pattern>
```

Comment:
Nice. I like that you've used `app-group` because if there were, say, 9 appendices and we specified `app`, then we'd get 9 warnings, when we only really need one.

Sometimes if there's just one appendix, typesetters will just include `<app>` inside of back, instead of as a child of `<app-group>`. WDYT to changing this context to `app-group|app[not(parent::app-group)]` to try to capture all possible cases (and not fire numerous unnecessary warnings).